docs/sdks/c-sharp/quickstart.md

```diff
@@ -270,42 +294,60 @@ void Reducer_OnSetNameEvent(ReducerEvent reducerEvent, string name)
+
+Then we can send our subscription to the SpacetimeDB module. SpacetimeDB is set up so that each client subscribes via SQL queries to some subset of the database, and is notified about changes only to that subset. For complex apps with large databases, judicious subscriptions can save each client significant network bandwidth, memory and computation compared. For example, in [BitCraft](https://bitcraftonline.com), each player's client subscribes only to the entities in the "chunk" of the world where that player currently resides, rather than the entire game world. Our app is much simpler than BitCraft, so we'll just subscribe to the whole database using `SubscribeToAllTables`.
 
 ```csharp
-void OnConnect()
+void OnConnected(DbConnection conn, Identity identity, string authToken)
 {
-    SpacetimeDBClient.instance.Subscribe(new List<string>
-    {
-        "SELECT * FROM User", "SELECT * FROM Message"
-    });
+    local_identity = identity;
+    AuthToken.SaveToken(authToken);
+
+    conn.SubscriptionBuilder()
+        .OnApplied(OnSubscriptionApplied)
+        .SubscribeToAllTables();
 }
 ```
 
-## OnIdentityReceived callback
+You can also subscribe to specific tables using SQL syntax, e.g. `SELECT * FROM my_table`. Our [SQL documentation](/docs/sql) enumerates the operations that are accepted in our SQL syntax.
 
-This callback is executed when we receive our credentials from the SpacetimeDB module. We'll use the `AuthToken` module to save our token to local storage, so that we can re-authenticate as the same user the next time we connect. We'll also store the identity in a global variable `local_identity` so that we can use it to check if we are the sender of a message or name change. This callback also notifies us of our client's `Address`, an opaque identifier SpacetimeDB modules can use to distinguish connections by the same `Identity`, but we won't use it in our app.
+## Connect Error callback
+
+Should we get an error during connection, we'll be given an `Exception` which contains the details about the exception. To keep things simple, we'll just write the exception to the console.
 
 ```csharp
-void OnIdentityReceived(string authToken, Identity identity, Address _address)
+void OnConnectError(Exception e)
 {
-    local_identity = identity;
-    AuthToken.SaveToken(authToken);
+    Console.Write($"Error while connecting: {e}");
+}
+```
+
+## Disconnect callback
+
+When Disconnecting, the callback contains the connection details and if an error occurs, it will also contain an `Exception`. If we get an error, we'll write the error to the console, if not, we'll just write that we disconnected.
```

Comment:
```suggestion
When disconnecting, the callback contains the connection details and if an error occurs, it will also contain an `Exception`. If we get an error, we'll write the error to the console, if not, we'll just write that we disconnected.
```

Either it's a symbol, in which case it gets code-formatted, or it's an English word, in which case it doesn't get capitalized.